engine/execution/computation/computer/computer_test.go

```diff
@@ -502,6 +502,10 @@ func (e *testRuntime) SetResourceOwnerChangeHandlerEnabled(_ bool) {
 	panic("SetResourceOwnerChangeHandlerEnabled not expected")
 }
 
+func (e *testRuntime) SetInvalidatedResourceValidationEnabled(_ bool) {
```

Comment:
This should go away if you pull from [janez/secure-cadence](https://github.com/onflow/flow-go/tree/janez/secure-cadence) again.